FinanceTracker/FinanceTracker/Program.cs

```diff
@@ -21,20 +21,32 @@
         menuSelection = userInput.ToLower();
     }
 
-    ICommand? command = null;
+    DisplayCommandInfo? display = new DisplayCommandInfo();
     switch (menuSelection)
     {
         case "1":
-            command = new AddIncomeCommand();
-            command.Execute(account);
+            ICommand<Operation> addIncomeCommand = new AddIncomeCommand();
+            account.UpdateBalance(addIncomeCommand.Execute(account));
```

Comment:
I think this should be executed inside command and you need to return result of this operation